src/main/java/edu/kis/vh/nursery/DefaultCountingOutRhymer.java

```diff
@@ -2,32 +2,35 @@
 
 public class DefaultCountingOutRhymer {
 
-    private int[] numbers = new int[12];
+    public static final int TAB_SIZE = 12;
```

Comment:
This constant is misnamed. You should specify that's max size or capacity of variable "numbers"